db/monitor/AOC2343.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<monitor name="AOC 2343" init="standard">
+	<caps add="(prot(monitor)type(lcd)model(AOC ID2008)cmds(01 02 03 07 0C 4E F3 E3)vcp(02 04 05 08 0B 0C 10 12 14(01 05 08 0B) 16 18 1A 60(01 03) 6C 6E 70 AC AE B6 C0 C6 C8 C9 CA CC( 00 02 03 04 05 08 09 0A 0D) D6(01 04) DF FF)mswhql(1)mccs_ver(2.1)asset_eep(32)mpu_ver(01))"/>
+
+	<!-- specific controls, but they might(?) be shared with other AOC monitors -->
```

Comment:
Probably they are just standard VESA (not) defined in: https://github.com/ddccontrol/ddccontrol-db/blob/master/db/monitor/VESA.xml

It needs a bit of studying of the standard, and some cleanup, though,..